### ISSUE TITLE
jQuery 1.9.0 and jQuery UI 1.10.0 Compatibility

### ISSUE THREAD

**Issue by gotqn** [fnagel/jquery-ui#286]
Is anyone working on jQuery 1.9.0  and jQuery UI 1.10.0 Compatibility?

**Comment by jedierikb** [comment]
duplicate of #273 ?

**Comment by fnagel** [comment]
Current version uses 1.9. 1.10 should be supported by the new, built from scratch version, see: #140

**Comment by gotqn** [comment]
finagel, thanks for informing me.  

It is said that "Selectmenu Widget is now scheduled to be released with jQuery UI 1.11".

**Comment by fnagel** [comment]
Yeah, thats right. Current selectmenu branch uses jQuery 1.9.0 and jQuery UI 1.10.0

**Comment by fnagel** [comment]
To make this more clear:

This branch uses jQuery 1.8.x and jQuery UI 1.9.x and the support of this old version will be dropped as soon as the new version (#140) is available.

If you need a version working with the latest jQuery / UI you should test the jQuery UI Selectmenu branch (using jQuery 1.9 and jQuery UI 1.10.1pre): https://github.com/jquery/jquery-ui/tree/selectmenu

**Comment by gotqn** [comment]
Thanks a lot. We are waiting for the new released.

**Comment by fnagel** [comment]
So this issue could be closed for now?

**Comment by gotqn** [comment]
Yes, thanks again.

**Comment by fnagel** [comment]
Reopened as this branch will be updated to work with jQuery 1.9.x and jQuery UI 1.10.x

Upgrade guide: http://jqueryui.com/upgrade-guide/1.9/

Hints: 
https://github.com/fnagel/jquery-ui/issues/273#issuecomment-13128389
